core/trino-main/src/main/java/io/trino/execution/TaskManagerConfig.java

```diff
@@ -78,6 +78,7 @@
     private Duration interruptStuckSplitTasksDetectionInterval = new Duration(2, TimeUnit.MINUTES);
 
     private boolean scaleWritersEnabled = true;
+    private int scaleWritersMaxNodesCount = 100;
```

Comment:
Why 100?

Reply:
Default value for `hashPartitionCount` is `100`. These options in some sense similar (in the sense of introducing a limit on number of nodes taking part in the query's stages). We need to decide on any value, the `100` seems to be reasonable choice.